### PR TITLE
Show chart when finish

### PR DESCRIPTION
**Pull request by tinvv**
Changes
- Show chart when finish
- Change input mode
- Add wordCount option

### CODE REVIEW

**Comment by ronnapatp** [comment]
Seem like in typescript chart.js can't export Chart

**Comment by narze** [comment]
@tinvv The wrong words should also have red background, and green on the correct ones. Use the color in the failing tests
image

Also, this PR is big and hard to review all the features added, would be appreciate if next time you can break them down to 1 feature per PR 🙏🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,34 +7,30 @@
 
 ## [1.1.2](https://github.com/Manoonchai/learn/compare/v1.1.1...v1.1.2) (2021-11-18)
 
-
 ### Bug Fixes
 
-* Allow pressing modifier keys e.g. refresh page ([b72db43](https://github.com/Manoonchai/learn/commit/b72db439dc76451143356c9e76cf5551764f1f7c))
+- Allow pressing modifier keys e.g. refresh page ([b72db43](https://github.com/Manoonchai/learn/commit/b72db439dc76451143356c9e76cf5551764f1f7c))
```
Comment (narze) [inline CHANGELOG.md:12]: This file should not be formatted since it was auto generated by the script

**Comment by narze** [comment]
Should be separate into many small PRs:
- Upgrade Tailwind to v3
- Render wpm graph with Chart.js
- Removal of text input (may not be necessary though)
- Spellcheck every character & highlight each character
- Add category separator in settings modal

**Pull request closed without merging**